01_05_Conditions.txt

```diff
@@ -9,100 +9,99 @@ anatomy that contian liquids vital to their survival. This can happen to both
 
-    Most Characters respond poorly to the application of extremely low 
+    Most characters respond poorly to the application of extremely low 
 temperatures to their anatomy. Biological targets tend to have the water in 
 their cells solidify into sharp ice in a process known as "Frostbite." 
 Cybernetic targets on the other hand tend to have their moving parts contract
 and wear down while their electrolytic fluid hardens. 
 
-    A Character that is Frozen takes d10 damage per round at the beginning of 
+    A character that is Frozen takes 1d10 damage per round at the beginning of 
 their turn. The Condition lasts until a source of heat is applied to the wound, 
 or with Medicine (Dex) Skill check, DC 50.
 
-== Fear ==
+== Fear (Afraid) ==
 
-    Both cybernetic and biologic Characters can react to the mental instinct for
+    Both cybernetic and biologic characters can react to the mental instinct for
 self-preservation. This usually happens when presented with the appearance of
 certain, agonizing death, though there are certainly other scary things in the
 universe.
 
-    A Character that has the Fear Condition may take no actions, except moving 
-away from the source of fear, or taking an action to roll to defeat fear. DC 50, 
-Will Save.
+    A character that is Afraid may take no actions, except moving away from
+the source of fear, or taking an action to roll a DC 50 Will Save to defeat Fear.
 
-== Fear, Greater ==
+== Fear, Greater (Afraid) ==
 
-    In rare cases, fear can become so great as to cause a total mental 
-overstimulation. In biologic Characters, this takes the form of a petrifying 
-flood of neurochemicals in the brain. Cybernetic Characters experience an 
+    In rare cases, Fear can become so great as to cause a total mental 
+overstimulation. In biologic characters, this takes the form of a petrifying 
+flood of neurochemicals in the brain. Cybernetic characters experience an 
 exception caused by the overinflation of their fear fuzzy logic field. 
 
-    A Character experienceing Greater Fear can take no actions, except rolling 
-to defeat fear. DC 75, Will save or 5 turns, whichever happens first.
+    A character experiencing Greater Fear can take no actions, except rolling 
+a DC 75 Will Save to defeat Fear. After 5 turns, Greater Fear ends regardless of
+the character's success or failure on the Will Save.
 
 == Lonely ==
 
     Being Lonely is a condition most commonly contracted from having 1 Charisma
 and failing to successfully socialize, or by trying and failing to socialize 
-with an ally who has 1 Charisma. A Lonely Character hasn't met Mazlow's 
+with an ally who has 1 Charisma. A Lonely character hasn't met Mazlow's 
 Hierarchy of needs, and this affects their performance. 
 
-    A Lonely Character moves 1m slower every two Actions (round down) and takes 
-a -5 to skill rolls. 
+    A Lonely character moves 1m slower every two Actions (round down) and takes 
+a -5 to Skill checks. 
 
 == Necrosis ==
```

Comment:
This is a definite rabbit hole, but we should totally make a Corrosion condition that does this for robotic species.